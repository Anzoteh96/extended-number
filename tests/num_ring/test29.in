-470773833011221523990544345606038586710357478211741619658646014098413413988368384149750
192659897763220357732707876492500017949281902481252042341616937673034066114090191956007
-126703606980527259127930045670127011297734675663906379047614312644035222188985316815688
