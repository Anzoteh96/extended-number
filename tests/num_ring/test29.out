Enter the number base b:
Enter two numbers n and m:
b is 470773833011221523990544345606038586710357478211741619658646014098413413988368384149750
n mod b is 192659897763220357732707876492500017949281902481252042341616937673034066114090191956007
m mod b is 344070226030694264862614299935911575412622802547835240611031701454378191799383067334062
n + m is 65956290782693098604777830822373006651547226817345663294002625028998843925104875140319
n - m is 319363504743747616860637922162627029247016578145158421389231250317069288303075508771695
n * m is 12453406509991418495788447303093442236655561004133150722604996505461023200316237584184
n ** m is 387114913019209827469521111793156034929699499470328745227578759747353544173193871793299
